.lando.yml

```diff
@@ -29,13 +29,16 @@ services:
       - if [ ! -e $LANDO_MOUNT/wp-cli.yml ]; then cp $LANDO_MOUNT/config/wp-cli.yml $LANDO_MOUNT/wp-cli.yml; fi
     run:
       # Setup WordPress for contributing via git.
+      # TODO: Rename core-git to core or core-dev.
       - if [ ! -e $LANDO_MOUNT/public/core-git ]; then git clone --depth 1 https://github.com/WordPress/wordpress-develop.git $LANDO_MOUNT/public/core-git; fi
+      - if [ ! -e $LANDO_MOUNT/public/core-git/.svn ]; then echo "Checking out WordPress SVN..."; cd $LANDO_MOUNT; svn co -q --ignore-externals https://develop.svn.wordpress.org/trunk/ tmp-svn; mv tmp-svn/.svn public/core-git/.svn; rm -rf tmp-svn; fi
```

Comment:
I think this `echo ...` notice should be removed. We don't have these for other processes. It might help to provide such information to make the output more helpful, but I'd prefer if we did that as a whole in a separate PR if necessary.

Reply:
The reason I included it is to give a reason for the apparent hanging of the process. The SVN checkout can take some time. So in removing the `echo` I'll also remove the `-q` so the downloaded files will populate the terminal.